docs/userguide/networking/get-started-overlay.md

```diff
@@ -308,41 +314,9 @@ to have external connectivity outside of their cluster.
 
 ## Step 6: Extra Credit with Docker Compose
 
-You can try starting a second network on your existing Swarm cluster using Docker Compose.
-
-1. If you haven't already, install Docker Compose.
-
-2. Change your environment to the Swarm master.
-
-		$ eval $(docker-machine env --swarm mhs-demo0)
-
-3. Create a `docker-compose.yml` file.
-
-4. Add the following content to the file.
-
-		web:
-			image: bfirsh/compose-mongodb-demo
-			environment:
-				- "MONGO_HOST=counter_mongo_1"
-				- "constraint:node==mhs-demo0"
-			ports:
-				- "80:5000"
-		mongo:
-			image: mongo
-
-5. Save and close the file.
-
-6. Start the application with Compose.
-
-		$ docker-compose --x-networking --project-name=counter up -d
-
-7. Get the Swarm master's IP address.
-
-		$ docker-machine ip mhs-demo0
-
-8. Put the IP address into your web browser.
-
-	Upon success, the browser should display the web application.
+Please refer to the Networking feature introduced in [Compose V2 format]
+(https://github.com/docker/compose/blob/master/docs/networking.md) and execute the
```

Comment:
Is this *page* going to be published on `docs.dockerproject.org` and if yes, shouldn't we link to the docs page instead of the github link ? :angel: